src/gulp-jsdoc.js

```diff
@@ -81,8 +89,6 @@ export function jsdoc(config, done) {
                     args = args.concat(['-t', inkdocstrap]);
                 }
 
-                args = args.concat(files);
```

Comment:
I am worried about the case where a user has a source defined and then expects additional files passed to also be documented. I haven't looked closely at this, but it seems that without ever doing args.concat(file) this case is impossible to work around it seems.

You do make a valid point that we should respect all or none of source. What do you think of if source was defined by user, keep old way of adding args?